src/legacy/ui/public/agg_types/controls/ranges.tsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment, useState, useEffect } from 'react';
+import {
+  htmlIdGenerator,
+  EuiButtonIcon,
+  EuiFieldNumber,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiIcon,
+  EuiSpacer,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { isEqual, omit } from 'lodash';
+import { AggParamEditorProps } from 'ui/vis/editors/default';
+
+const generateId = htmlIdGenerator();
+const isEmpty = (value: any) => value === undefined || value === null;
```

Comment:
As long as you're using lodash helpers, why not use the equivalent helper to this?

Reply:
Not sure which lodash helper you are talking about, but if you mean `isNil` - it is only available from the version `4.0.0`. (currently we are using `3.10.1`)